src/build_targets.rs

```diff
@@ -326,4 +328,27 @@ mod test {
 
         assert_eq!(file_names.unwrap(), expected);
     }
+
+    #[test]
+    fn cygwin() {
+        let target = Target {
+            is_target_overridden: false,
+            arch: String::from(""),
+            os: String::from("cygwin"),
+            env: String::from(""),
+            cfg: Vec::new(),
+            target: None,
+        };
+        let file_names = FileNames::from_target(&target, "ferris", Path::new("/foo/bar"), false);
+
+        let expected = FileNames {
+            static_lib: PathBuf::from("/foo/bar/libferris.a"),
+            shared_lib: PathBuf::from("/foo/bar/ferris.dll"),
```

Comment:
```suggestion
            shared_lib: PathBuf::from("/foo/bar/cygferris.dll"),
```

No?

Reply:
As far as I understood it that's the filename that rustc creates, before it gets copied to the right name via cinstall, and I don't know where I could change that (??). Same as with mingw currently.

So this breaks things.